web/components/templates/sessions/sessionId/SessionContentV2.tsx

```diff
@@ -0,0 +1,186 @@
+import { useRouter } from "next/router";
+import { useMemo, useState } from "react";
+import { Session } from "../../../../lib/sessions/sessionTypes";
+import { useLocalStorage } from "../../../../services/hooks/localStorage";
+import { useGetRequests } from "../../../../services/hooks/requests";
+import { Col } from "../../../layout/common/col";
+import getNormalizedRequest from "../../requestsV2/builder/requestBuilder";
+import RequestDrawerV2 from "../../requestsV2/requestDrawerV2";
+import { BreadCrumbV2 } from "./breadCrumbV2";
+import ChatSession from "./Chat/ChatSession";
+import TreeView from "./Tree/TreeView";
+import { Row } from "@/components/layout/common";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs";
+import { Button } from "@/components/ui/button";
+import { EyeIcon, EyeOffIcon } from "lucide-react";
+
+interface SessionContentV2Props {
+  session: Session;
+  session_id: string;
+  requests: ReturnType<typeof useGetRequests>;
+}
+
+export const TABS = [
+  {
+    id: "tree",
+    label: "Tree",
+  },
+  {
+    id: "chat",
+    label: "Chat",
+  },
+] as const;
+
+const SessionContentV2: React.FC<SessionContentV2Props> = ({
```

Comment:
This is fine, but in the future, we want to deprecate the idea of "V2" and just change the original, as GIT handles versioning for us. Totally understand you were following how we previously did it.